components/navbar/navbar.jsx

```diff
@@ -647,7 +647,9 @@ export default class Navbar extends React.Component {
                             </li>
                         </ChannelPermissionGate>
                     );
+                }
 
+                if (!ChannelStore.isDefault(channel)) {
```

Comment:
Can we add test snapshots of `createDropdown` with non-default and default channels?

Reply:
Sure :)

Reply:
Done!